### DISCUSSION TITLE
How would I connect multiple accounts at once

### DISCUSSION THREAD

**Discussion by DoggersFor1** [PrismarineJS/mineflayer#2437] [Q&A]
is there way to connect multiple accounts at once using a object or something? also is there a discord?

**Comment by amoraschi** [comment]
You can join Discord by clicking one of the badges in the Readme. This question has been asked multiple times, I think there are other discussions similar to this one

We also have 2 examples in the examples folder: multiple.js and multiple_from_file.js

**Comment by DoggersFor1** [comment]
Thanks, will take a look!
